docs/integrations/app-integrations/youtube.mdx

```diff
@@ -5,66 +5,81 @@ sidebarTitle: YouTube
 
 In this section, we present how to connect YouTube to MindsDB.
 
-[YouTube](https://www.youtube.com/) is a popular online video-sharing platform and social media website where users can upload, view, share, and interact with videos created by individuals and organizations from around the world.
+[YouTube](https://www.youtube.com/) is a popular online video-sharing platform and social media website where users
+can upload, view, share, and interact with videos created by individuals and organizations from around the world.
 
-Data from YouTube can be utilized within MindsDB to train AI models and chatbots based on user comments, detect sentiment of comments, or use AI models to create replies.
+Comment Data from YouTube can be utilized within MindsDB to train AI models and chatbots based on user comments,
+detect sentiment of comments, or use AI models to create replies.
 
 ## Connection
 
-This handler is implemented using the `google-api-python-client` library, a Python library that from where we can use YouTube v3 API.
+First, we have to install the required python packages.
+```bash
+pip install mindsdb[youtube]
+```
+
+## Usage
```

Comment:
Please remove this "## Usage" from this line.

The Connection section ends after the CREATE DATABASE statement. And there is already the Usage section.